src/index.ts

```diff
@@ -104,7 +104,7 @@ const main = async (argv: string[]): Promise<any> => {
   }
   switch (args._[0]) {
     case "batch":
-      if ("ni" in args) {
+      if ("nonInteractive" in args) {
```

Comment:
this block is not needed anymore you can always do `nonInteractive: options.nonInteractive`